src/www/ui/api/Models/Analysis.php

```diff
@@ -69,6 +70,11 @@ class Analysis
    * Whether to schedule ojo agent or not
    */
   private $ojo;
+  /**
+   * @var boolean $pkgagent
```

Comment:
Oops, the doc was updated for the variable.

Reply:
well spotted ;-)